src/components/scenes/GuiPluginListScene.js

```diff
@@ -10,13 +10,13 @@ import AntDesignIcon from 'react-native-vector-icons/AntDesign'
 import { updateOneSetting } from '../../actions/SettingsActions.js'
 import { COUNTRY_CODES, FLAG_LOGO_URL } from '../../constants/CountryConstants.js'
 import { customPluginRow, guiPlugins } from '../../constants/plugins/GuiPlugins.js'
-import { PLUGIN_VIEW } from '../../constants/SceneKeys.js'
+import { PLUGIN_VIEW_DEEP } from '../../constants/SceneKeys.js'
```

Comment:
I think this should stay `PLUGIN_VIEW`, and we should just delete the whole `PLUGIN_VIEW_DEEP` constant, replacing it with `PLUGIN_VIEW` where it appears.